class.jetpack-settings.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+/**
+ * All the Settings we want to sync to .com
+ *
+ * This is also used when displaying settings in the API
+ */
+class Jetpack_Settings {
```

Comment:
You mentioned that the name of this is a bit generic, and has potential for confusion. What about something like `Jetpack_Sync_Settings`?